react-events/my-app/src/CustomButton.tsx

```diff
@@ -0,0 +1,16 @@
+type Props = {
+  text: string;
+  color: string;
+  onCustomClick: (text: string) => void;
+};
+
+export default function CustomButton(props: Props) {
+  function handleClick() {
+    props.onCustomClick(props.text);
+  }
+  return (
+    <button style={{ backgroundColor: props.color }} onClick={handleClick}>
```

Comment:
You can now use the event handler you received as a prop directly.

`onClick={() => props.onCustomClick(props.text)}`